use-cases/inferencing/serving/vllm/hyperdiskML/README.md

```diff
@@ -0,0 +1,309 @@
+# Distributed Inferencing on vLLM using HyperdiskML
+
+This guide demonstrates how to serve a model with vllm using hyperdiskML. By the end of this guide, you should be able to perform the following steps:
+
+- Create a hyperdiskML for the LLM model weights
+- Deploy a vLLM container to your cluster to host your model
+- Use vLLM to serve the fine-tuned Gemma model
+- View Production metrics for your model serving
+- Use custom metrics and Horizontal Pod Autoscaler (HPA) to scale your model
+
+## Prerequisites
+
+- This guide was developed to be run on the [playground AI/ML platform](/platforms/gke-aiml/playground/README.md). If you are using a different environment the scripts and manifest will need to be modified for that environment.
+- A bucket containing the fine-tuned model from the [Fine-tuning example](/use-cases/model-fine-tuning-pipeline/fine-tuning/pytorch/README.md)
+
+## Preparation
+
+- Clone the repository
+
+  ```sh
+  git clone https://github.com/GoogleCloudPlatform/accelerated-platforms && \
+  cd accelerated-platforms
+  ```
+
+- Change directory to the guide directory
+
+  ```sh
+  cd use-cases/inferencing/serving/vllm/hyperdiskML
```

Comment:
Update path based on comment above.

Reply:
The path `use-cases/inferencing/serving/vllm/hyperdiskML` seems to be correct to me, it is inline with other guides

Reply:
This was in reference to the comment:

Can you change this to `hyperdiskml`, `hyperdisk-ml`, `hdml` for consistency of not using camel case.